src/RosBagDataProvider.cpp

```diff
@@ -120,15 +120,12 @@ bool RosbagDataProvider::spin() {
         // Publish VIO output if any.
         // TODO(Toni) this could go faster if running in another thread or
         // node...
-        bool got_synced_outputs = publishSyncedOutputs();
```

Comment:
Where did the behavior of publishSyncedOutputs go to? At some point we need to publish the pose, mesh, time horizon points, etc

Reply:
spinOnce in RosVisualizer.cpp
spinOnce in RosDisplay.cpp